bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -1085,6 +1085,62 @@ Number of dominant landunits, so this determines the number of active landunits.
 <default>Default: 0</default>
 </entry>
 
+<entry id="toosmall_soil" type="integer" category="physics"
+       group="clm_inparm"
+       valid_values="0,1,2,3,4,5,6,7,8,9,10" value="0">
+Threshold above which the model keeps the soil landunit. Selecting the value 0 means DO NOTHING. Valid values reach 100 but practical values are not expected above 10 at this time.
```

Comment:
What is the reason for limiting from 0 to 10? I suppose it makes sense to limit to integers, rather than real, but that's artificial as well.

Reply:
Somewhere in here and in the code, I think it would be good to say that this is a percentage of the landunits in the gridcell. It gives the range as 0 to 100, which implies it's a percentage, but it doesn't explicitly say that. It would be helpful to have the documention talk about it being a percentage.

Reply:
I added the word "Percentage" in the front of the comments for each of these namelist parameters. I added similar reference to the corresponding comment in surfrdMod.F90.

My comments here state:
"Valid values reach 100 but practical values are not expected above 10 at this time."
However, would it work if I changed what I have to valid_values="0 to 100"? I don't know how valid_values is used and whether that would still work @ekluzek 

Regarding reals vs. integers, do you prefer that I change these to reals?

Reply:
Something that we could start doing is including the units in all parameter and variable names. For example these would become toosmall_*_percent

What do you think?

Reply:
Now I know that setting `valid_values="0 to 100"` would not work, so I will write out every integer from 0 to 100.

Reply:
@slevisconsulting we talked about this in our CLM software meeting, and actually we don't think you should limit it to integers for the namelist input. We think you should allow for double precision floating point input between 0.0 and 100.0.

We aren't sure that it's that useful to input say 9.5 rather than 9, but didn't see why to limit to integers.

Reply:
Ok, I will just replace "integer" with "real" from what I see elsewhere in the file and will remove `valid_values`.

Reply:
...and will modify elsewhere in the code accordingly.

Reply:
Committed change of the toosmalls from integer to real.